src/pages/blocks/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react'
+import { parseAge } from '../../utils'
+import { useWhaleApiClient } from '@contexts/WhaleContext'
+import { AdaptiveTable } from '@components/commons/AdaptiveTable'
+import { Link } from '@components/commons/Link'
+
+export interface Block {
+  id: string
+  hash: string
+  previousHash: string
+
+  height: number
+  version: number
+  time: number
+  medianTime: number
+
+  transactionCount: number
+
+  difficulty: number
+
+  masternode: string
+  minter: string
+  minterBlockCount: number
+
+  stakeModifier: string
+  merkleroot: string
+
+  size: number
+  sizeStripped: number
+  weight: number
+}
```

Comment:
You do not need to declare an interface here. You can import the block interface from whale directly. 

```suggestion
import { Block } from '@defichain/whale-api-client/dist/api/blocks'

```

Reply:
Removed the duplication interface definition